gke-kafka-strimzi/terraform/modules/network-mirror/main.tf

```diff
@@ -12,7 +12,7 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-# [START vpc_multi_region_network]
+# [START vpc_streaming__kafka_strimzi_multi_region_network]
```

Comment:
Each tag in this sample should start with `gke_streaming_kafka_strimzi_`, so this one could be, for example, `gke_streaming_kafka_strimzi_vpc_multi_region_network`

Same comment for the other region tags

Reply:
fixed